### PR TITLE
commit

### PR DESCRIPTION
**Pull request by anastasia-radchencko**
https://anastasia-radchencko.github.io/react_carousel/

### CODE REVIEW

File: src/components/Carousel.tsx [hunk 1]
```diff
@@ -1,18 +1,107 @@
...
+          {'<<'}
+        </button>
+
+        <button
+          className={cn(
+            'button',
+            { 'button--disabled': (isEnd && !infinite) },
+          )}
+          type="button"
+          onClick={() => handleNextImage()}
```
Comment (etojeDenys) [inline src/components/Carousel.tsx:98]:
```suggestion
          onClick={handleNextImage}
```
we can simplify it

**Review by etojeDenys — CHANGES_REQUESTED** [review]
let's add some min and max limits to inputs
image

File: src/components/Carousel.tsx [hunk 2]
```diff
@@ -1,18 +1,107 @@
...
+      <div className="buttons">
+        <button
+          type="button"
+          className={cn(
+            'button',
+            { 'button--disabled': (isStart && !infinite) },
+          )}
+          onClick={() => handlePrevImage()}
+        >
+          {'<<'}
```
Comment (VitaliyBodnar) [inline src/components/Carousel.tsx:89]:
```suggestion
          <<
```

File: src/components/Carousel.tsx [hunk 3]
```diff
@@ -1,18 +1,107 @@
...
+      </ul>
+
+      <div className="buttons">
+        <button
+          type="button"
+          className={cn(
+            'button',
+            { 'button--disabled': (isStart && !infinite) },
+          )}
+          onClick={() => handlePrevImage()}
```
Comment (VitaliyBodnar) [inline src/components/Carousel.tsx:87]:
```suggestion
          onClick={handlePrevImage}
```

File: src/components/Carousel.tsx [hunk 4]
```diff
@@ -1,18 +1,107 @@
...
+
+        <button
+          className={cn(
+            'button',
+            { 'button--disabled': (isEnd && !infinite) },
+          )}
+          type="button"
+          onClick={handleNextImage}
+        >
+          {'>>'}
```
Comment (VitaliyBodnar) [inline src/components/Carousel.tsx:100]:
```suggestion
          >>
```

File: src/App.tsx [hunk 5]
```diff
@@ -22,18 +22,82 @@ class App extends React.Component<{}, State> {
     ],
   };
 
-  render() {
-    const { images } = this.state;
+  const { images } = state;
+  const [sizeImages, setSizeImages] = useState(130);
+  const [step, setStep] = useState(3);
+  const [frameSize, setFrameSize] = useState(3);
+  const [animationDuration, setAnimationDuration] = useState(1000);
```
Comment (VitaliyBodnar) [inline src/App.tsx:29]: make sense to move theses default values to enum, as it will make a code cleaner

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
- all tests are not passed 
Screenshot 2023-09-04 at 15 20 12

File: src/components/Carousel.tsx (hunk 4 above)
Comment (anastasia-radchencko) [reply src/components/Carousel.tsx:100]: image

**Pull request closed without merging**